addon/components/flexberry-dropdown.js

```diff
@@ -154,6 +153,15 @@ export default FlexberryBaseComponent.extend({
     }
   })),
 
+  /**
+    Flag indicates whether to show input with search class.
+
+    @property isSearch
+    @type Boolean
+    @default false
+  */
+  isSearch: false,
```

Comment:
Base class? It is flexberry-dropdown component. For mode search we need to have input with search class. Otherwise semantic-ui remove input from DOM when we select some value and we can't select other value.